goreq.go

```diff
@@ -325,6 +327,14 @@ func (r Request) Do() (*Response, error) {
 		})
 	}
 
+	if r.ShowDebug {
+		dump, err := httputil.DumpRequest(req, true)
+		if err != nil {
+			println(err.Error())
```

Comment:
Can you change `println` to `fmt.Println`. We usually prefer not to use bootstrapped (https://golang.org/ref/spec#Bootstrapping) stuff as they're not standard.